pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -94,6 +94,13 @@ func (oe openshiftExtension) PostReconcile(ctx context.Context, comp v1alpha1.Te
 			return err
 		}
 	}
+
+	if configInstance.Spec.Profile == common.ProfileBasic {
+		addon, _ := extension.GetAddon(oe.operatorClientSet.OperatorV1alpha1().TektonAddons(), common.DashboardResourceName)
+		if addon != nil {
+			return extension.TektonAddonCRDelete(oe.operatorClientSet.OperatorV1alpha1().TektonAddons(), common.AddonResourceName)
```

Comment:
Don't we require deletion of triggers here for lite profile :thinking:

Reply:
Yes it requires, have updated it!